src/libraries/src.proj

```diff
@@ -17,5 +21,23 @@
     <Project Include="$(DirectoryToBuild)\src\*.*proj" />
   </ItemGroup>
 
-  <Import Project="$(MSBuildThisFileDirectory)dir.traversal.targets" />
+  <Target Name="IncludeProjectReferences">
+    <ItemGroup>
+      <ProjectReference Include="@(Project)" />
+    </ItemGroup>
+  </Target>
+
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
+  
+  <Target Name="Build" DependsOnTargets="BuildAllProjects" />
```

Comment:
It might be worth it to add a comment above Build target to say that ordering matters and that we are intentionally defining this after importing sdk.targets to override that value, that way if people in the future edit this file will keep that in mind.